src/logical/builder_proxy.rs

```diff
@@ -13,16 +13,17 @@ use crate::{
 
 /// Trait capturing builder proxies that use plain string (used for parsing in logical layer)
 pub trait LogicalColumnBuilderProxy<'a, 'b>: ColumnBuilderProxy<String> {
-    /// create a new [`LogicalColumnBuilderProxy`] which nests its physical target equivalent [`BuilderProxy`][crate::physical::builder_proxy::PhysicalBuilderProxyEnum].
+    /// Create a new [`LogicalColumnBuilderProxy`] which nests its physical target equivalent [`BuilderProxy`][crate::physical::builder_proxy::PhysicalBuilderProxyEnum].
```

Comment:
It's not important that some nesting happens here – what is important is that this can be constructed from a given `PhysicalBuilderProxyEnum`.